README.md

```diff
@@ -80,9 +40,11 @@ The Docker image for these notebooks is hosted in the [planetlabs/notebooks](htt
 cd notebooks
 docker pull planetlabs/notebooks
 docker tag planetlabs/notebooks planet-notebooks
-```
 
-If you get errors while running the Docker commands above, you might have to add `sudo` at the beginning, such as `sudo docker pull planetlabs/notebooks`.
+# If you get errors running the above, you might have to add sudo to the beginning:
+#sudo docker pull planetlabs/notebooks
+#sudo docker tag planetlabs/notebooks planet-notebooks
```

Comment:
instead of adding commented out lines inside of one example code block, you could add a separate code block

Reply:
This actually hit me -- I originally had some text below this code block mentioning the sudo workaround, but had "documentation blindness" as I was focused on just the docker commands themselves. I put the commented out sudo portions inside that code block so that someone who is trying to run those commands would quickly see that they might have to add sudo depending on their user account and how docker was installed.